packages/devtools_shared/lib/src/directory_picker/scripts.dart

```diff
@@ -0,0 +1,17 @@
+const macScript = r'''
+#!/bin/bash
+
+# Copyright 2018 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Fast fail the script on failures.
+set -ex
+
+osascript -e '
+  tell application (path to frontmost application as text)
```

Comment:
This is a applescript to launch a directory picker with title `Select a Flutter project` to be on top of front most application